components/SockJsOrderClient.js

```diff
@@ -0,0 +1,133 @@
+import React, {Component} from 'react'
+import {apiRoot} from "../constants/Backend";
+import SockJsClient from 'react-stomp';
+import * as Device from 'expo-device';
+import {playSound} from "../helpers/playSoundHelper";
+
+
+const url = `${apiRoot}/ws`
+
+class SockJsClientOrderDisplayBase extends Component {
+  constructor(props, context) {
+    super(props, context)
+  }
+
+  render() {
+    const {
+      meta,
+      topics,
+      handleSockClientData,
+      handleConnect,
+      handleDisConnect,
+      clientId,
+      debugMode,
+    } = this.props;
+
+
+    return (
+      <SockJsClient url={url} topics={[`${topics}${clientId}`]}
+        onMessage={(data) => {
+          if (data?.needAlert) {
+            playSound()
+          }
+          handleSockClientData(data)
+        }}
+        ref={(client) => {
+          this.clientRef = client
+        }}
+        onConnect={() => {
+          handleConnect()
+          console.log(`onConnect: ${Device.osName}-${Device.osVersion}`)
+        }}
+        onDisconnect={() => {
+          handleDisConnect()
+          console.log('onDisconnect')
+        }}
+        debug={debugMode}
+      />
+    );
+  }
+};
+export const SockJsClientOrderDisplay = SockJsClientOrderDisplayBase
+
+
+class SockJsClientTableDisplayBase extends Component {
```

Comment:
I believe the user of this component should instead use the RealTimeOrderUpdate component.